site/docs/transform/window.md

```diff
@@ -101,3 +101,16 @@ Rather than filtering the above two examples we can also calculate a residual by
 We can also use `rank` operator to calculate ranks over time.
 
 <div class="vl-example" data-name="window_rank"></div>
+
+### Top K
+
+Here we use window transform to derive the total number of students along with the rank of the current student to determine the top K students and display their score.
+
+<div class="vl-example" data-name="window_top_k"></div>
+
+
+### Running Average
```

Comment:
This is "Cumulative Running Average", not just random moving average, isn't it? 

Maybe a bit clearer?  

https://en.wikipedia.org/wiki/Moving_average#Cumulative_moving_average